clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -41,34 +34,52 @@ def api_call_latency(title, verb, scope, threshold):
         yAxes=g.single_y_axis(format=g.SECONDS_FORMAT),
     )
 
+ApiCallLatencyPanelArgs = namedtuple('ApiCallLatencyPanelArgs', ['title', 'verb', 'scope', 'threshold'])
 
-CLUSTERLOADER_PANELS = [
-    api_call_latency(
+API_CALL_LATENCY_PANELS_TEMPLATE = [
+    ApiCallLatencyPanelArgs(
         title="Read-only API call latency (percentaile=99, scope=resource, threshold=1s)",
         verb="GET",
         scope="namespace",
-        threshold=1,
+        threshold=1
     ),
-    api_call_latency(
+    ApiCallLatencyPanelArgs(
         title="Read-only API call latency (percentaile=99, scope=namespace, threshold=5s)",
         verb="LIST",
         scope="namespace",
         threshold=5,
     ),
-    api_call_latency(
+    ApiCallLatencyPanelArgs(
         title="Read-only API call latency (percentaile=99, scope=cluster, threshold=30s)",
         verb="LIST",
         scope="cluster",
         threshold=30,
     ),
-    api_call_latency(
+    ApiCallLatencyPanelArgs(
         title="Mutating API call latency (threshold=1s)",
         verb=d.any_of("CREATE", "DELETE", "PATCH", "POST", "PUT"),
         scope=d.any_of("namespace", "cluster"),
         threshold=1,
     ),
 ]
 
+API_CALL_LATENCY_PANELS = [api_call_latency(title=n.title, verb=n.verb, scope=n.scope, threshold=n.threshold, expression="""
+apiserver:apiserver_request_latency_1m:histogram_quantile{
+  quantile="0.99",
+  verb=~"%(verb)s",
+  scope=~"%(scope)s",
+  resource=~"${resource:regex}s*",
```

Comment:
I should have add this earlier, but could you extract panel generation to a function e.g. `def api_call_latency_panel(expression)`? I think this would be cleaner that setting so many const in this file.

Reply:
Added function.